src/nxdoc/web-ui/nuxeo-elements/nuxeo-elements-tutorial.md

```diff
@@ -29,10 +29,10 @@ Watch the related courses on Nuxeo University:</br>
 
 ## Requirements
 
-- **[Node.js](https://nodejs.org/)** is a JavaScript runtime built on Chrome's V8 JavaScript engine. Almost every tool out there for client-side development is built with Node.js and distributed with npm, the package manager for Node.js. Make sure you download and install your OS-specific version first (last verified versions for this tutorial are node v10.12.0 and npm 6.4.1)
+- **[Node.js](https://nodejs.org/)** is a JavaScript runtime built on Chrome's V8 JavaScript engine. Almost every tool out there for client-side development is built with Node.js and distributed with npm, the package manager for Node.js. Make sure you download and install your OS-specific version first, according to the requirements described in the [Nuxeo Elements repository](https://github.com/nuxeo/nuxeo-elements/blob/master/README.md#dependencies).
```

Comment:
I think we should also link to the README.md#dependencies for both Elements and Web UI somewhere else more visible. A suggestion would be [here](https://doc.nuxeo.com/nxdoc/nuxeo-elements/) and [here](https://doc.nuxeo.com/nxdoc/web-ui-overview/).

Reply:
Sure, I'll do that.